src/aspire/basis/fb.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import numpy as np
+
+from aspire.basis.basis_utils import num_besselj_zeros
+from aspire.utils.matlab_compat import m_reshape
+
+logger = logging.getLogger(__name__)
+
+
+class FBBasisMixin(object):
+    """
+    FBBasisMixin is a mixin implementing methods specific to Fourier-Bessel expansions,
+    to be inherited by Fourier-Bessel subclasses of Basis.
+    """
+
+    def _getfbzeros(self):
+        """
+        Generate zeros of Bessel functions
+        """
+        # get upper_bound of zeros of Bessel functions
+        upper_bound = min(self.ell_max + 1, 2 * self.nres + 1)
+
+        # List of number of zeros
+        n = []
+        # List of zero values (each entry is an ndarray; all of possibly different lengths)
+        zeros = []
+
+        # generate zeros of Bessel functions for each ell
+        for ell in range(upper_bound):
+            # for each ell, num_besselj_zeros returns the zeros of the
+            # order ell Bessel function which are less than 2*pi*c*R = nres*pi/2,
+            # the truncation rule for the Fourier-Bessel expansion
+            _n, _zeros = num_besselj_zeros(
```

Comment:
I think we discussed in #591 renaming this function to something like `all_besselj_zeros` since it doesn't just count, but returns all the zeros.